Classes/Domain/Validator/ElectronicAddressValidator.php

```diff
@@ -65,8 +66,10 @@ public function isValid($value)
             } else {
                 $result = $addressValidator->validate($value->getIdentifier());
                 if ($result->hasErrors()) {
+                    $identifierResult = $this->getResult()->forProperty('identifier');
                     foreach ($result->getErrors() as $error) {
-                        $this->addError($error->getMessage(), $error->getCode());
+                        /** @var Error $error */
```

Comment:
I'd move this before the `foreach`, so it's closer to where the variable first appears.

Reply:
Yes looks better. Is fixed ;)